Content.Shared/_RMC14/Xenonids/Parasite/SharedXenoParasiteSystem.cs

```diff
@@ -592,6 +594,8 @@ public override void Update(float frameTime)
 
                 var knockdownTime = infected.BaseKnockdownTime * 75;
                 InfectionShakes(uid, infected, knockdownTime, infected.JitterTime, false);
+                var mod = new PainModificator(knockdownTime, infected.StrongChestBurstPain, PainModificatorType.PainIncrease);
+                _pain.AddPainModificator(uid, mod, pain);
```

Comment:
Make this a method and call it, you are repeating it in every section of this system